sedlex.opam

```diff
@@ -14,6 +14,7 @@ depends: ["jbuilder" {build}
           "ppx_tools_versioned"
           "ocaml-migrate-parsetree"
           "gen"
+          "uchar"
```

Comment:
That's only a dependency on the older versions of course.

Reply:
Is there a way to express that on opam? Also, isn't the compatibility library a no-op on newer versions?

Happy to fix, just not sure how to usefully change it.